### PR TITLE
PLAT-869 Doc fixes

### PR DESCRIPTION
**Pull request by emosesPatreon**
**PLAT-869 Doc fixes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This will provide you with a `client_id` and a `client_secret`.
 Let's say you wanted to make a "Log In with Patreon" button.
 You've read through [the directions](https://www.patreon.com/platform/documentation/oauth),
 and are trying to implement "Step 2: Handling the OAuth Redirect" with your server.
-The user will be arriving at one of your pages *after* you have sent them to [the authorize page](www.patreon.com/oauth2/authorize) for step 1,
+The user will be arriving at one of your pages *after* you have sent them to the authorize page (at https://www.patreon.com/oauth2/authorize) for step 1,
```
Comment (emosesPatreon) [inline README.md]: Actually clicking on that link is useless.

File: README.md [hunk 2]
```diff
@@ -62,15 +62,16 @@ $pledge = null;
  * Given a resource in the document, you can access its relationships with ->relationship->($relationship_name)
  * or check for a relationship's existence with ->has('relationships.relationship_name')
  */
-if ($user->has('relationships.pledges')) {
+if ($patron->has('relationships.pledges')) {
     /*
      * To look up the full resource that the relationship is referencing,
-     * we have extended [art4/json-api-client/ResourceIdentifier](https://github.com/Art4/json-api-client/blob/master/docs/objects-resource-identifier.md)
+     * we have extended art4/json-api-client/ResourceIdentifier
```
Comment (emosesPatreon) [inline README.md]: Links are formatted inside code blocks, so this was hard to read.

File: README.md (hunk 1 above)
Comment (spencerhandley) [reply README.md]: Good point

File: README.md (hunk 2 above)
Comment (spencerhandley) [reply README.md]: 👍

**Pull request merged**